src/Components/App/App.js

```diff
@@ -4,21 +4,63 @@ import Movies from '../Movies/Movies'
 import movieData from '../../movieData'
 import Header from '../Header/Header'
 import Footer from '../Footer/Footer'
+import MovieDetails from '../MovieDetails/MovieDetails'
 
 class App extends Component {
   constructor() {
     super()
-    this.state = { movies: movieData }
+    this.state = { 
+      movies: movieData.movies,
+      currentMovie: ''
+     }
   }
+
+  handleClick = (id) => {
+    const current = this.state.movies.find(movie => movie.id === id)
+    this.setState({currentMovie: current})
+  }
+
+  exitDetails = () => {
+    this.setState({currentMovie: ''})
+  }
+
   render() {
     return (
-      <div className="App">
+      <div className='App'>
         <Header />
-        <Movies movies={this.state.movies}/>
+
+        {!this.state.currentMovie && (
+        <Movies movies={this.state.movies} handleClick={this.handleClick}/>
+        )}
+
+        {this.state.currentMovie && (
+        <MovieDetails currentMovie={this.state.currentMovie} exitDetails={this.exitDetails}/>
```

Comment:
I don't think I've seen the parentheses around the JSX in the docs/lessons that I've seen so far so I'm curious to see if they're necessary, but glad that this works!